components/clp-mcp-server/tests/server/test-utils.py

```
@@ -0,0 +1,130 @@
+"""Unit tests for CLP MCP server's utility functions."""
```

Ruff (0.14.1)

File `components/clp-mcp-server/tests/server/test-utils.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)